timemachine/cpp/src/wrap_kernels.cpp

```diff
@@ -1219,7 +1234,8 @@ void declare_barostat(py::module &m) {
             py::arg("frequency"),
             py::arg("bps"),
             py::arg("seed"),
-            py::arg("adaptive_scaling_enabled") = true)
+            py::arg("adaptive_scaling_enabled") = true,
+            py::arg("initial_volume_scale_factor") = 0.0)
```

Comment:
possible doc or line comment to pre-empt worry: "0.0" is a flag meaning "use 1% of first box volume" ( https://github.com/proteneer/timemachine/pull/1152#discussion_r1333509394 )